### PR TITLE
plugin.video.fox.news 6.0.1

### PR DESCRIPTION
**Pull request by learningit**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.fox.news/resources/lib/scraper.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+h = HTMLParser.HTMLParser()
+qp  = urllib.quote_plus
+uqp = urllib.unquote_plus
+UTF8 = 'utf-8'
+
+
+class myAddon(t1mAddon):
+
+  def getAddonMenu(self,url,ilist):
+      ilist = self.addMenuItem('Live','GV', ilist, '5614615980001', self.addonIcon, self.addonFanart, {}, isFolder=False)
```
Comment (enen92) [inline plugin.video.fox.news/resources/lib/scraper.py:24]: please localize the menu items

File: plugin.video.fox.news/resources/lib/scraper.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      ilist = self.addMenuItem('Show Clips','GM', ilist, 'show', self.addonIcon, self.addonFanart, {}, isFolder=True)
+      ilist = self.addMenuItem('News Clips','GM', ilist, 'news', self.addonIcon, self.addonFanart, {}, isFolder=True)
+      return(ilist)
+
+
+
+  def getAddonMovies(self,url,ilist):
+      html = self.getRequest('https://video.foxnews.com')
```
Comment (enen92) [inline plugin.video.fox.news/resources/lib/scraper.py:32]: probably move the url to a common variable? It's used in more than one method

File: plugin.video.fox.news/addon.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            library="default.py">
+        <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Fox News Videos</summary>
+    <description lang="en_GB">Fox News Videos</description>
+    <disclaimer lang="en_GB">Feel free to use this script. For information visit the wiki.</disclaimer>
+    <platform>all</platform>
+    <language>en</language>
+    <license>LGPL-2.0-or-later</license>
```
Comment (enen92) [inline plugin.video.fox.news/addon.xml:20]: I don't think the license file you are adding is LGPL, please confirm

Comment (learningit) [reply plugin.video.fox.news/addon.xml:20]: Not sure, isn't it? 
https://spdx.org/licenses/LGPL-2.0-or-later.html

Is there a difference between a GNU Public License and a GNU Library public license?
I

Comment (learningit) [reply plugin.video.fox.news/addon.xml:20]: NM. I found the right one.
https://spdx.org/licenses/GPL-2.0-or-later.html

File: plugin.video.fox.news/resources/lib/scraper.py (hunk 1 above)
Comment (learningit) [reply plugin.video.fox.news/resources/lib/scraper.py]: done

File: plugin.video.fox.news/resources/lib/scraper.py (hunk 2 above)
Comment (learningit) [reply plugin.video.fox.news/resources/lib/scraper.py]: done

File: plugin.video.fox.news/addon.xml (hunk 3 above)
Comment (learningit) [reply plugin.video.fox.news/addon.xml]: done

**Pull request merged**